k9mail/src/main/java/com/fsck/k9/preferences/GlobalSettings.java

```diff
@@ -348,18 +348,18 @@
      * Upgrades the settings from version 23 to 24.
      *
      * <p>
-     * Set <em>messageViewTheme</em> to {@link K9.Theme#USE_GLOBAL} if <em>messageViewTheme</em> has
+     * Set <em>message_view_theme</em> to {@link K9.Theme#USE_GLOBAL} if <em>message_view_theme</em> has
      * the same value as <em>theme</em>.
      * </p>
      */
     private static class SettingsUpgraderV24 implements SettingsUpgrader {
 
         @Override
         public Set<String> upgrade(Map<String, Object> settings) {
-            K9.Theme messageViewTheme = (K9.Theme) settings.get("messageViewTheme");
+            K9.Theme messageViewTheme = (K9.Theme) settings.get("message_view_theme");
             K9.Theme theme = (K9.Theme) settings.get("theme");
             if (theme != null && messageViewTheme != null && theme == messageViewTheme) {
-                settings.put("messageViewTheme", K9.Theme.USE_GLOBAL);
+                settings.put("message_view_theme", K9.Theme.USE_GLOBAL);
```

Comment:
Same goes for all changes in GlobalSettings: I don't think changing them here is correct!